npm-packages/meteor-installer/README.md

```diff
@@ -1,16 +1,54 @@
 ## Meteor Installer
 
-Node.js <=14.x and npm <=6.x is recommended.
+### Recommended Versions
 
-Install Meteor by running:
+- For Meteor 2
+  - Use Node.js 14.x or higher
+  - Use npm 6.x or higher
+- For Meteor 3
+  - Use Node.js 20.x or higher
+  - Use npm 9.x or higher
+
+### Installation
+
+To install Meteor, run the following command:
 
 ```bash
 npm install -g meteor
 ```
 
-[Read more](https://www.meteor.com/developers/install)
+#### Running with `npx`:
+
+```bash
+npx meteor
+
+# or with a specific version
+npx meteor@<version>
+```
```

Comment:
I think this is the preferred way to install Meteor, since `npm install -g meteor` doesn't provide any feedback logs unless you do: `npm install -g meteor --foreground-script` . May we just keep then npx option here, or at least switch the order?

Reply:
I think it would be better to keep only one option.

Reply:
I agree, I will invert the order on those then

Reply:
I didn't see your comment there @fredmaiaarantes , well I will go with the simplest one then `npx`.